src/routes/search.py

```diff
@@ -25,27 +26,31 @@ def get_blueprint():
 
 
 @search_blueprint.route("/search_ingredients", methods=["POST"])
-def search_ingredients():
-    if flask.request.method == "POST":
-        ingredients = flask.request.form.get("searchIngredients")
+def searchByingredients():
+    if request.method == "POST":
+        ingredients = request.form.get("searchIngredients")
         # prints what the user searches for to the terminal
         print(ingredients)
-        returnedDict = api.spoonacular.search_ingredients(ingredients)
+        returnedDict = search_ingredients(ingredients)
         # prints the 10 recipes to the terminal
         print(returnedDict)
+        if returnedDict == []:
+            print("error")
         return render_template("index.html", ingredients=returnedDict)
     return render_template("index.html")
 
 
 @search_blueprint.route("/search_recipes", methods=["POST"])
-def search_recipes():
-    if flask.request.method == "POST":
-        ingredients = flask.request.form.get("searchRecipes")
+def searchByrecipes():
+    if request.method == "POST":
+        ingredients = request.form.get("searchRecipes")
         # prints what the user searches for to the terminal
         print(ingredients)
-        returnedDict = api.spoonacular.search_recipes(ingredients)
+        returnedDict = search_recipes(ingredients)
         # prints the 10 recipes to the terminal
         print(returnedDict)
+        if returnedDict == []:
+            print("error")
         return render_template("index.html", recipes=returnedDict)
     return render_template("index.html")
```

Comment:
See above comment regarding return for reference as well as the error print message